project/AppDependencies.scala

```diff
@@ -2,27 +2,28 @@ import sbt._
 
 object AppDependencies {
 
-  val silencerVersion = "1.7.7"
+  val silencerVersion = "1.7.9"
 
   val compile: Seq[ModuleID] = Seq(
     "uk.gov.hmrc"     %% "play-partials"                % "8.2.0-play-28",
     "uk.gov.hmrc"     %% "url-builder"                  % "3.5.0-play-28",
     "uk.gov.hmrc"     %% "tax-year"                     % "1.6.0",
-    "uk.gov.hmrc"     %% "bootstrap-frontend-play-28"   % "5.24.0",
+    "uk.gov.hmrc"     %% "bootstrap-frontend-play-28"   % "7.2.0",
     "uk.gov.hmrc"     %% "play-frontend-hmrc"           % "3.17.0-play-28",
```

Comment:
this ticket was to upgrade play-frontend-hmrc to latest version (3.24.0-play-28)